app/store/message.go

```diff
@@ -159,3 +159,16 @@ func GetMessageById(id int64) (error, *Message) {
 	}
 	return nil, &message[0]
 }
+
+// GetMessageByTimestamp returns  a message that is found by it's timestamp
+func GetMessageByTimestamp(timestamp uint64) (*Message, error) {
```

Comment:
You have a `findQuotedMessage` with the query `SELECT * FROM messages WHERE sentat = ?` and now you have a `GetMessageByTimestamp` but with `SELECT * FROM messages WHERE outgoing = 1 AND sentat = `. Can the name of the methods be `findMessage` and `findOutgoingMessage`?

Reply:
I agree with the `findOutgoingMessage` but the other one depends on `*signalservice.DataMessage_Quote` so it only works in that context.